android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/AuthorizationsSchema.java

```diff
@@ -9,7 +9,7 @@
     String COLUMN_AUTHORIZATIONS_ID = "id"; // type: int
     String COLUMN_AUTHORIZATIONS_IDENTITY_ID = "identity_id"; // type: long
     String COLUMN_AUTHORIZATIONS_ISSUED = "issued"; // type: long
-    String COLUMN_AUTHORIZATIONS_PUBLIC_KEY_ID = "public_key_id"; // type: long
+    String COLUMN_AUTHORIZATIONS_ACCOUNT_ID = "public_key_id"; // type: long
```

Comment:
Is `COLUMN_AUTHORIZATIONS_ACCOUNT_ID` value suppose to be `"public_key_id"`? It's unchanged from the previous schema.

Reply:
yes, good callout. this made it easy(easier) to migrate, as I don't need to migrate this column. This column will be changed again anyways when we move to multi account.

Reply:
ah ya know, now that I think about this more, I might not be handling migration of these ids correctly...

I am only migrating the publickey records over to the new AccountRecord but I do not actually update these ids. hmm. will get this fixed asap

Reply:
Summarizing my understanding of the issue:

Pre-migrated databases will have an Authorizations Table with rows that contain a column (`public_key_id`) that contains a `publicKeyId`. When the migration step (`onUpgrade`) occurs, the Authorizations Table is not updated and the `public_key_id` column will still contain `publicKeyIds` rather than `accountIds`.

This is problematic because certain parts of code will assume that the column contains `accountIds` when they are actually still `publicKeyIds`. I'll add comments for spots where I see this being problematic.


Solutions:
1. Create a new column for `account_id` in Authorizations Table then populate it during the migration step `onUpgrade`. Migrated DBs will just have a redundant `public_key_id` column that isn't used anymore.

2. On upgrade/migration, alter each row in Authorizations Table so that `public_key_id` == `account_id`, by editing/updating each `public_key_id` value. (Is it possible to change a row's id? Where is the row's id even determined?).